defi/src/protocols/data.ts

```diff
@@ -2989,7 +2989,7 @@ export default [
     logo: `${baseIconsUrl}/fei-protocol.png`,
     audits: "2",
     audit_note: null,
-    gecko_id: "fei-protocol",
+    gecko_id: "fei-usd",
```

Comment:
this should be `tribe-2` instead

Reply:
[TRIBE](https://www.coingecko.com/en/coins/tribe) and [FEI](https://www.coingecko.com/en/coins/fei-usd) have their own entity on coingecko. Don't you think it will be better to add a dedicated entry for TRIBE @0xngmi ?